scripts/generate_driver_wrappers.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""This script is required for the auto generation of the
+   psa_crypto_driver_wrappers.c file"""
+
+import sys
+import os
+import jinja2
+
+def render(template_path: str) -> str:
+    environment = jinja2.Environment(
+        loader=jinja2.FileSystemLoader(os.path.dirname(template_path)),
```

Comment:
This nested invocation can throw quite a few exceptions. It could be useful to catch them and notify the user as to why.

Reply:
What would be the point of catching them? What notification could we offer that's better than Python's default trace?

Reply:
Actually I do see one thing we should improve about error reporting, which is to indicate where Jinja is looking for files. This is the error when it can't find the template:
```
Traceback (most recent call last):
  File "/home/gilpes01/z/dev/mbedtls/reviewing/5193-codegen_1.0/library/../scripts/generate_driver_wrappers.py", line 30, in <module>
    RESULT = render(DRIVER_WRAPPER_TEMPLATE_FILENAME)
  File "/home/gilpes01/z/dev/mbedtls/reviewing/5193-codegen_1.0/library/../scripts/generate_driver_wrappers.py", line 13, in render
    template = environment.get_template(os.path.basename(template_path))
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/lib/python3/dist-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/lib/python3/dist-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/lib/python3/dist-packages/jinja2/loaders.py", line 187, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: psa_crypto_driver_wrappers.conf
```
Because we're passing the basename and the directory separately to Jinja, the error trace doesn't indicate the directory.